carsus/io/nist/ionization.py

```diff
@@ -154,6 +154,12 @@ def parse_ground_level(row):
             except KeyError:
                 pass
 
+            # To handle cases where the ground level J has not been understood:
+            # Take as assumption J=0
+            # Consider adding a log or warning message???
```

Comment:
I think adding a warning message would be a really good idea. Maybe something along these lines could work:
```
print(f"Set J=0 for ground state of species ({row['atomic_number']}, {row['ion_charge']}).")
```

Reply:
I forgot about this. IIRC there are 147 missing `J`'s betweet `Pr` to `U`, so the logging would be very long. But I don't see a better solution.

Reply:
Shall I go ahead and add Christian's suggested line, then? I can do that easily enough.

Reply:
Yes, do it.

Reply:
Ok - I've added that now - and it seems to work ok! Have re"pushed" - so hopefully that shows up properly here now too! Thanks!